### ISSUE TITLE
COUNT DISTINCT error

### ISSUE THREAD

**Issue by eddieajau** [oguimbal/pgsql-ast-parser#12]
Bumped into this error:

```
const { parseFirst } = require('pgsql-ast-parser');

const ast = parseFirst(`select count(distinct(id)) from some_table`);
```

Error:

```
Error: Syntax error at line 1 col 14:

  select count(distinct
               ^
Unexpected kw_distinct token: "distinct". Instead, I was expecting to see one of the following:

    - A "int" token
    - A "rparen" token
...
```

**Comment by oguimbal** [comment]
Thanks for the report. I'll try to fix it asap.

**Comment by eddieajau** [comment]
@oguimbal thanks for the quick reply. If you could walk me through the logic sometime I can perhaps report the next one as a PR :)

**Comment by oguimbal** [comment]
Sure, [this](https://github.com/oguimbal/pg-mem/blob/master/CONTRIBUTING.md#a-implement-its-parser) should guide you a bit :)

If you try, and find yourself stuck, dont hesitate to ask for help !

**Comment by eddieajau** [comment]
@oguimbal nice. I'll look at that the commit you make to fix and hopefully that will save you some work "next time". Thanks!

**Comment by oguimbal** [comment]
Fixed with `pgsql-ast-parser@4.1.13` :)

**Comment by eddieajau** [comment]
@oguimbal thankyou. Confirmed fixed.
